content/kb/troubleshooting-some-old-instance-manager-pods-are-still-running-after-upgrade.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Troubleshooting: Some old instance manager pods in longhorn-system are still running after upgrade"
```

Comment:
I think we could drop the wording `longhorn-system`. [1]

Reply:
I prefer to add `Extended reading: .....` to link to the KB in live-upgrade section. How about it?

Reply:
Sure